tools/engine_tool/test/build_command_test.dart

```diff
@@ -53,8 +52,9 @@ void main() {
       cannedProcesses: cannedProcesses,
     );
     try {
-      final List<Build> result = runnableBuilds(testEnv.environment, configs, true);
-      expect(result.length, equals(8));
+      final List<Build> result =
+          runnableBuilds(testEnv.environment, configs, true);
+      expect(result.length, equals(4));
```

Comment:
Could you expand why this went down to 4? (I'll keep reviewing, maybe it'll become obvious later in review)

Reply:
<img width="778" alt="Screenshot 2024-08-28 at 4 24 09 PM" src="https://github.com/user-attachments/assets/b0cc66df-fe4c-4167-b9b6-502dcd50acd5">

This change (I don't think we cared about duplicate builds here, it was just copy pasta-d)

Reply:
Ah nice!